README.md

```diff
@@ -30,6 +30,11 @@ New users, you don't have to worry about this - just follow the instructions in
 ### Prerequisites
 
 #### First, clone the repo to your computer, then navigate to the starfish folder and run `npm install`.
```

Comment:
This should be npm ci, not npm install

Reply:
I've made this change in the PR I'm about to push up. Feel free to change it here too, but no worries if you don't.